src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -279,19 +301,24 @@ public ContentAccessCertificate getCertificate(Consumer consumer) {
     private ContentAccessCertificate createNewScaCertificate(Consumer consumer, Owner owner)
         throws IOException, GeneralSecurityException {
         log.info("Generating new SCA certificate for consumer: \"{}\"", consumer.getUuid());
-        Validity oneYearValidity = Validity.oneYear();
-        CertificateSerial serial = createSerial(oneYearValidity);
+        OffsetDateTime start = OffsetDateTime.now().minus(1, ChronoUnit.HOURS);
```

Comment:
java.time classes have a set of convenience methods for time updates such as `minusHours` or `plusYears`.

Reply:
Updated